Doc/library/stdtypes.rst

```diff
@@ -4659,11 +4659,12 @@ other sequence-like behavior.
 
 There are currently two built-in set types, :class:`set` and :class:`frozenset`.
 The :class:`set` type is mutable --- the contents can be changed using methods
-like :meth:`~set.add` and :meth:`~set.remove`.  Since it is mutable, it has no
-hash value and cannot be used as either a dictionary key or as an element of
-another set.  The :class:`frozenset` type is immutable and :term:`hashable` ---
-its contents cannot be altered after it is created; it can therefore be used as
-a dictionary key or as an element of another set.
+like :meth:`add <frozenset.add>` and :meth:`remove <frozenset.add>`.
```

Comment:
There is no `frozenset.add()`.

Reply:
Because `set` and `frozenset` are documented together, there is no cross-reference target for `set.add`, it is labelled as `frozenset.add` instead. This link goes to the correct place.

Reply:
I agree. I plan to fix it, but it requires changes (& a release) of Sphinx. This PR makes the reference link to the right place, and more importantly means that no future reference errors can be introduced into `stdtypes.rst`.

Once it is fixed, I plan to go back and update here & a few other places where we have used a different target to the intended one.

Reply:
#141857 implements this approach.